test/fantasy-land/results.js

```diff
@@ -1,11 +1,24 @@
+import Absolute from './index.js';
+
 export default [
-  [
-    'uses Z.equals for equality checks',
-    [
-      true,
-      'Absolute (-1)',
-      'Absolute (1)',
-      2,
-    ],
-  ],
+  {
+    description: 'uses Z.equals for equality checks',
+    doctest: {
+      correct: true,
+      input: {
+        lines: [
+          {number: 1, text: '> Absolute(-1)'},
+        ],
+        throws: false,
+        result: Absolute (-1),
+      },
+      output: {
+        lines: [
+          {number: 2, text: 'Absolute(1)'},
+        ],
+        throws: false,
+        result: Absolute (1),
+      },
+    },
+  },
```

Comment:
We now have access to the results as values rather than just their string representations. :)